deploy_scripts/start.sh

```diff
@@ -3,24 +3,25 @@
 
 set -euo pipefail
 
-set +u
-source /home/ec2-user/.bash_profile
-set -u
+# copy repo to /var/app/flashcrow
+sudo rm -rf /var/app/flashcrow
+sudo cp -R ~ec2-user/flashcrow /var/app
 
-# copy private config to repo
-cp -r /home/ec2-user/flashcrow.config.js /home/ec2-user/flashcrow/lib/config/private.js
+# copy private / forever configs to /var/app/flashcrow
+sudo chown -R ec2-user:ec2-user /var/app/flashcrow
+cp -r /home/ec2-user/flashcrow.config.js /var/app/flashcrow/lib/config/private.js
+cp /var/app/flashcrow/scripts/deployment/web/forever.json /var/app/flashcrow/forever.json
```

Comment:
there's actually no reason to do this - we can simplify by just passing `/var/app/flashcrow/scripts/deployment/web/forever.json` as the `forever` config below.